src/ol/layer/MapboxVector.js

```diff
@@ -197,9 +197,11 @@ const SourceType = {
  * @property {number} [minResolution] The minimum resolution (inclusive) at which this layer will be
  * visible.
  * @property {number} [maxResolution] The maximum resolution (exclusive) below which this layer will
- * be visible.
+ * be visible. If neither `maxResolution` nor `minZoom` are defined, the layer's `maxResolution` will
+ * match the source's `minZoom`
```

Comment:
```suggestion
 * match the source's `minZoom`.
```

only if you're already making other changes